src/tutorials/0008-merkle-dags/08.md

```diff
@@ -0,0 +1,31 @@
+---
+    title: "Merkle DAGs as a building block"
+    type: "text"
+---
+
+Merkle DAGs give us a flexible way to model and share data on the
+distributed web. This is a fundamental capability&mdash;so much so, in
+fact, that Merkle DAGs are the basic building block for many
+diverse projects: version control systems like Git, blockchains
+like Ethereum, decentralized web protocols like IPFS, and
+distributed storage networks like Filecoin all use Merkle DAGs to
+store and communicate data!
+
+This widespread use speaks to a potential ability to use Merkle
+DAGs as a common basis of communication between disparate
+projects. That’s exactly the idea behind the InterPlanetary
```

Comment:
Should we hyperlink here?

Reply:
Both! I updated this in my latest batch of edits.